bin/start-pgbouncer-as-service

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+# vim: ft=bash
+
+set +e
+
+export PGBOUNCER_PROCESS_REGEX="[v]endor/pgbouncer/bin/pgbouncer"
+export PGBOUNCER_BUILD_PACK_PID
+
+function inf() {
+  echo -e "INFO:  $* "
+}
+
+function err() {
+   echo -e "ERROR: $* "
+}
+
+function is-enabled() {
+  ( shopt -s extglob nocasematch
+    [[ $1 == @(1|true|yes|on) ]]
+  )
+}
+
+function pgbouncer-current-pid() {
+  /bin/ps -eopid,args | grep -E "${PGBOUNCER_PROCESS_REGEX}" | awk '{print $1}'
+}
+
+function is-pgbouncer-running() {
+  local current_pid
+  current_pid=$(pgbouncer-current-pid)
```

Comment:
You could just define this on the same line like
```bash
local current_pid=$(pgbouncer-current-pid)
```
or is there a reason to split them apart?